integrations/access/accessrequest/app.go

```diff
@@ -136,9 +167,25 @@ func (a *App) run(ctx context.Context) error {
 	return nil
 }
 
+func (a *App) checkIfAMRIsRelevent(amr *accessmonitoringrulesv1.AccessMonitoringRule) bool {
```

Comment:
Can we think of a better name for this method? I'm not sure what "is relevant" is supposed to mean.

Reply:
Is `amrAppliesToThisPlugin` better?